benchmark/tinysegmenter3.py

```diff
@@ -13,7 +13,6 @@
 # https://github.com/chezou/nltk/blob/master/jpbook/tinysegmenter.py
 
 import re
-from benchmarker import Benchmarker
 try:
```

Comment:
These ```try...except``` block is for `benchmarker` compatibility.
Could you move them into after `import Benchmarker` clause?